www/src/examples/Form/Basic.js

```diff
@@ -1,19 +1,23 @@
 <Form>
-  <Form.Group controlId="formBasicEmail">
-    <Form.Label>Email address</Form.Label>
-    <Form.Control type="email" placeholder="Enter email" />
+  <div className="mb-3">
```

Comment:
Guess this gives us a good excuse to revive the [Layout](https://github.com/react-bootstrap/react-bootstrap/issues/3029) component idea 😉 Would be fantastic if we could get this implemented for V5